Core/include/Acts/Clusterization/NewClusterization.hpp

```diff
@@ -0,0 +1,61 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
```

Comment:
Nice, obviously we will have to change the title of the file ... can we just name the algorithm?

Reply:
Hi Andi, indeed, I was waiting until we could decide what to do with the old one before tackling the naming issue. Probably I should bring that up a the next dev meeting.

Reply:
I named the filed HKClusterization.hpp/ipp. Let me know if that does not fly...

Reply:
So following the naming guidelines we agreed on at some point in the past, we would prefer `HkClusterization`. Is that ok?

Reply:
@paulgessinger no issues with that